Userland/Services/WebContent/PageHost.cpp

```diff
@@ -121,7 +121,8 @@ Gfx::Color PageHost::background_color() const
 
 void PageHost::paint(Web::DevicePixelRect const& content_rect, Gfx::Bitmap& target)
 {
-    Gfx::Painter painter(target);
+    auto bitmap = Gfx::Bitmap::create(Gfx::BitmapFormat::BGRA8888, content_rect.size().to_type<int>()).release_value_but_fixme_should_propagate_errors();
+    Gfx::Painter painter(bitmap);
```

Comment:
This looks unused? 

The most drive-by review ever btw. I didn't look at any file but this one :D

Reply:
indeed! thank you, I removed it 😅